dotnet/src/Microsoft.AutoGen/Agents/Services/AgentWorkerHostingExtensions.cs

```diff
@@ -13,25 +11,9 @@ namespace Microsoft.AutoGen.Agents;
 
 public static class AgentWorkerHostingExtensions
 {
-    public static WebApplicationBuilder AddAgentService(this WebApplicationBuilder builder, bool local = false, bool useGrpc = true)
+    public static IHostApplicationBuilder AddAgentService(this IHostApplicationBuilder builder, bool local = false, bool useGrpc = true)
     {
-        if (local)
-        {
-            //TODO: make configuration more flexible
-            builder.WebHost.ConfigureKestrel(serverOptions =>
```

Comment:
I think it would be better to simply let user configure kestrel? It's not necessary for in-memory scenario

Reply:
yeah this is going away in my next pr